### PR TITLE
Improve documentation/test of -e

### PR DESCRIPTION
**Pull request by erikbosch**
Recent change for CLI change how `-e` must be used, like:

- Old: `-e a1,a2`
- New: `-e a1 -e a2`

As this is a backward incompatible change it might be good to give a warning. This PR assumes valid extra arguments ever contain a comma, and adds some tests

### CODE REVIEW

**Comment by sschleemilch** [comment]
#378 introduces that already: https://github.com/COVESA/vss-tools/pull/378/files#diff-b6edd36e3b26ecd6c94d19d1a0484fce6cbb7f88d11330368b6b55666afb5daeR37

**Review by sschleemilch — CHANGES_REQUESTED** [review]
Duplicate #378

**Comment by erikbosch** [comment]
> Duplicate #378

Greats minds thinks alike. I will review that one now!

**Comment by sschleemilch** [comment]
> > Duplicate #378
> 
> Greats minds thinks alike. I will review that one now!

Just merged master, we had merge conflits with #379.

**Comment by erikbosch** [comment]
Putting this one on hold for now. I can confirm that repeated `-e` works in the other PR. The test cases from this PR and some of the documentation updates might be relevant to keep, I will visit it again after the other is merged.

**Comment by erikbosch** [comment]
@sschleemilch - reworked this one now. I think it could be useful to give an error if someone use the old syntax with comma

File: docs/vspec.md
Comment (sschleemilch) [inline docs/vspec.md:320]: maybe use an [alert](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)

File: src/vss_tools/vspec/vssexporters/utils.py
Comment (sschleemilch) [inline src/vss_tools/vspec/vssexporters/utils.py:56]: The better place would be to do that directly in the [cli_options.py](https://github.com/COVESA/vss-tools/blob/master/src/vss_tools/vspec/cli_options.py) where the argument is defined.

We can write a function and define it for the `type=`:

```python
from click import option

def no_comma_type(value):
    if "," in value:
        raise click.BadParameter(f"'{value}' should not contain ','")

extend_all_attributes_opt = option(
    "--extend-all-attributes/--no-extend-all-attributes",
    help="""
        Generates all extended attributes,
        not only the ones given via '-e/--extended-attributes'
    """,
    default=False,
    show_default=True,
    type=no_comma_type
)
```

**Review by sschleemilch — CHANGES_REQUESTED** [review]

**Review by sschleemilch — APPROVED** [review]

**Pull request merged**
